src/options.rs

```diff
@@ -629,7 +631,15 @@ pub fn conn_params(tmp: &ConnectionOptions) -> anyhow::Result<Builder> {
         task::block_on(bld.read_dsn(dsn))?;
         bld.read_extra_env_vars()?;
     } else if let Some(instance) = &tmp.instance {
-        task::block_on(bld.read_instance(instance))?;
+        if instance.contains("/") {
```

Comment:
Shouldn't this PR also change the instance name parsing part of the Rust client before merging? Right now, for the name with format like `org_name/instance_name` CLI will complain about invalid characters for the name.

I think the problem is here: https://github.com/edgedb/edgedb-rust/blob/6b385b3c9f8c548fddc61cd53c1182c13c92ded7/edgedb-client/src/builder.rs#L296

Reply:
Yes, the client validation should be updated too. `<org>/<instance>` should be automatically treated as a Cloud instance.